src/IIIFPresentation/API.Tests/Integration/DeleteCollectionTests.cs

```diff
@@ -0,0 +1,236 @@
+    {
+        // Arrange
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/{RootCollection.Id}");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        var errorResponse = await response.ReadAsPresentationResponseAsync<Error>();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        errorResponse!.ErrorTypeUri.Should()
+            .Be("http://localhost/errors/DeleteCollectionType/CannotDeleteRootCollection");
+        errorResponse.Detail.Should().Be("Cannot delete a root collection");
+    }
+
+
+    [Fact]
+    public async Task DeleteCollection_FailsToDeleteCollection_WhenAttemptingToDeleteRootDirectly()
+    {
+        // Arrange
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/{RootCollection.Id}");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        var errorResponse = await response.ReadAsPresentationResponseAsync<Error>();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        errorResponse!.ErrorTypeUri.Should()
+            .Be("http://localhost/errors/DeleteCollectionType/CannotDeleteRootCollection");
+        errorResponse.Detail.Should().Be("Cannot delete a root collection");
+    }
+
+    [Fact]
+    public async Task DeleteCollection_FailsToDeleteCollection_WhenAttemptingToDeleteCollectionWithItems()
+    {
+        // Arrange
+        var deleteRequestMessage = HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete,
+            $"{Customer}/collections/FirstChildCollection");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(deleteRequestMessage);
+
+        var errorResponse = await response.ReadAsPresentationResponseAsync<Error>();
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        errorResponse!.ErrorTypeUri.Should().Be("http://localhost/errors/DeleteCollectionType/CollectionNotEmpty");
+        errorResponse.Detail.Should().Be("Cannot delete a collection with child items");
+    }
+    
+    [Fact]
+    public async Task DeleteIiifCollection_DeletesInS3()
+    {
+        // Arrange
+        var slug = nameof(DeleteIiifCollection_DeletesInS3);
+        var collection = new PresentationCollection
+        {
+            Behavior = new List<string>()
+            {
+                Behavior.IsPublic
+            },
+            Parent = RootCollection.Id,
+            Slug = slug
+        };
+
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Post, $"{Customer}/collections",
+                collection.AsJson());
+
+        var response = await httpClient.AsCustomer(1).SendAsync(requestMessage);
```

Comment:
It looks like all of these tests make requests as customer 1 - would it be worth defining a `customerId` const?